### PR TITLE
Fix some layout issues with logs tab

### PR DESCRIPTION
**Pull request by jshaughn**
Fixes https://github.com/kiali/kiali/issues/2816

And other things...
- remove splitPercent as it's only for initial positioning of the splitter
- use explicit keys for the different orientations to avoid the toggle issue
  - let each Splitter apply the proper attributes
- improve "NoLogs" message strings
- improve help tooltip
- fix a bunch of spacing and resize issues
  - splitter height compensates for toolbar height
  - allow splitter panes to compensate for splitter bar and/or title heights
  - adjust splitter max/mins
  - set splitter mins to %s not fixed values
  - adjust some margins and padding
- alphabetical ordering the large number of state variables and styles

### CODE REVIEW

**Comment by lucasponce** [comment]
Can you post a screenshot ?
Just to check that is aligned with other CSS issues we fixed recently.

**Comment by jshaughn** [comment]
@lucasponce I'll post a screenshot after this PR is revised, after rebasing on pending feature work from Mike and then some changes resulting from the most recent UX call.

**Comment by jshaughn** [comment]
@lucasponce @pbajjuri20 Here is the promised video that shows the CSS consistent with the other tabs, the primary fix, and also generally better-behaving layout.

kiali#2816

File: src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx [hunk 1]
```diff
@@ -148,19 +150,18 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
     if (this.props.pods.length < 1) {
       this.state = {
         duration: retrieveDuration() || 600,
+        hideLogValue: '',
```
Comment (mtho11) [inline src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx:153]: Aha! I see you like things alphabetical :)
I was wondering why there we so many things changed but I guess some of it is just lines moving around.

File: src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx [hunk 2]
```diff
@@ -399,6 +367,72 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
     );
   }
 
+  private getSplitter = () => {
```
Comment (mtho11) [inline src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx:370]: I like seeing this broken up as it is cleaner as it was getting big.

**Comment by pbajjuri20** [comment]
Thanks, @jshaughn. Layout issues fixed perfectly

Tested and passed Kiali UI 1.20.0 (b665f793830fb4f387ad75a0da4b096d677b48b5)

File: src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx (hunk 1 above)
Comment (jshaughn) [reply src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx:153]: +1, I am a little OCD on the alphabetizing

File: src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx (hunk 2 above)
Comment (jshaughn) [reply src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx:370]: It also helped set up to use different props for the different orientations.

**Pull request merged**